packages/kununu-theme-v3/scss/base.scss

```diff
@@ -1,6 +1,14 @@
 @import './variables.scss';
 @import './mixins.scss';
 
+/* Smooths font rendering for Chrome-based browsers and Firefox */
+body,
+html {
+  -moz-font-feature-settings: 'kern'; // stylelint-disable-line property-no-vendor-prefix
```

Comment:
This might not be needed https://developer.mozilla.org/en-US/docs/Web/CSS/font-feature-settings

Reply:
I will compare how it looks on Firefox with and without it